app/item.rb

```diff
@@ -1,7 +1,9 @@
 class Item
-  attr_accessor :source, :lable, :genre, :author
+  attr_accessor :source, :label, :genre, :author, :id, :publish_date
 
-  def initialize(publish_date, archived)
+  @bol = true
+
+  def initialize(publish_date, archived = @bol)
```

Comment:
nice making the default to true but let's make the default to false here since we will start with an archived Item.and toggle from there. we might want to do something like this
 `def initialize(publish_date, archived = false)`

Reply:
id is private so you might want to not put it in attr_accessor